.travis.yml

```diff
@@ -10,10 +10,9 @@ services:
   - docker
 
 install:
-  # gocov
-  - go get github.com/axw/gocov/gocov
-  # protobuf
-  - go get -v -d google.golang.org/grpc
+  - go get -v || (sleep 30 && go get -v) # Install dependencies listed in the go.mod and go.sum files (twice if flaky network).
+  - go get github.com/axw/gocov/gocov # Install gocov package.
+  - go get -v -d google.golang.org/grpc # Install protobuf package.
```

Comment:
This comment applies to the next few lines

Reply:
Done.